### PR TITLE
Release 1.8

### PR DESCRIPTION
**Pull request by pwittrock**
Fix version issues with 1.8 reference docs.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5680)

### CODE REVIEW

File: docs/api-reference/v1.8/index.html [hunk 1]
```diff
@@ -81863,24 +81579,21 @@ <h1 id="replicaset-v1beta2-apps">ReplicaSet v1beta2 apps</h1>
 </thead>
 <tbody>
 <tr>
-<td>apps</td>
-<td>v1beta2</td>
+<td>extensions</td>
```
Comment (chenopis) [inline docs/api-reference/v1.8/index.html]: Wait, isn't ReplicaSet supposed to be apps, v1beta2?

File: docs/api-reference/v1.8/index.html [hunk 2]
```diff
@@ -68955,20 +68783,20 @@ <h1 id="controllerrevision-v1beta2-apps">ControllerRevision v1beta2 apps</h1>
 <tbody>
 <tr>
 <td>apps</td>
-<td>v1beta2</td>
+<td>v1beta1</td>
```
Comment (chenopis) [inline docs/api-reference/v1.8/index.html]: This supposed to go from v1beta2 to v1beta1?

**Comment by steveperry-53** [comment]
Thank you @pwittrock. This looks good to me. Let's get @chenopis to chime in.

File: docs/api-reference/v1.8/index.html [hunk 3]
```diff
@@ -71712,21 +71480,21 @@ <h1 id="daemonset-v1beta2-apps">DaemonSet v1beta2 apps</h1>
 </thead>
 <tbody>
 <tr>
-<td>apps</td>
-<td>v1beta2</td>
+<td>extensions</td>
+<td>v1beta1</td>
 <td>DaemonSet</td>
```
Comment (chenopis) [inline docs/api-reference/v1.8/index.html]: Isn't DaemonSet supposed to be apps, v1beta2?

Comment (steveperry-53) [reply docs/api-reference/v1.8/index.html]: I think these are OK as is. (I think) this corrects the entries under OLD API VERSIONS. So the old API group version really should be extensions v1beta1.

Comment (chenopis) [reply docs/api-reference/v1.8/index.html]: Ah, ok.

**Comment by chenopis** [comment]
Yeah, looks good in the preview. /lgtm

**Comment by steveperry-53** [comment]
@pwittrock @chenopis OK to merge?

File: docs/api-reference/v1.8/index.html (hunk 3 above)
Comment (pwittrock) [reply docs/api-reference/v1.8/index.html]: I think it is: https://deploy-preview-5680--kubernetes-io-vnext-staging.netlify.com/docs/api-reference/v1.8/#daemonset-v1beta2-apps

**Pull request merged**